protocols/kad/src/behaviour.rs

```diff
@@ -1684,13 +1693,25 @@ where
 
             QueryInfo::GetClosestPeers { key, mut step } => {
                 step.last = true;
+                let peers = result
+                    .peers
+                    .map(|peer_id| {
+                        let addrs = result
+                            .inner
+                            .addresses
+                            .remove(&peer_id)
+                            .expect("peer not present in address map")
```

Comment:
We probably don't want to panic here
```suggestion
                            .unwrap_or_default()
```

Reply:
Right, I fixed it in the ok case, but forgot about this one.